tests/utils/lighthouse-criteria.ts

```
@@ -12,28 +12,28 @@ export interface PerformanceCriteria {
-  
+
   // Best Practices
   bestPracticesCompliant: boolean;
 }
 
 export class LighthouseCriteriaValidator {
   private distPath: string;
-  
-  constructor(distPath: string = './dist') {
+
+  constructor(distPath: string = "./dist") {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)